build/repo.targets

```diff
@@ -1,17 +1,8 @@
 <Project>
-  <Import Project="FixPackageOutputDirs.targets" />
-
   <UsingTask
     TaskName="PullRequestSubmitter.PullRequestTask"
     AssemblyFile="$(MSBuildThisFileDirectory)..\tools\PullRequestSubmitter\bin\Release\netcoreapp2.0\PullRequestSubmitter.dll" />
 
-  <Target Name="PushNuGetPackages">
```

Comment:
Are you removing this only because it's not needed in the patch branch? This wouldn't be removed when you make the same changes to `dev`, would it?

Reply:
Yeah, not needed in the patch branch. aspnet/Universe will push the packages to our patch myget feed.